apps/frontend/packages.d.ts

```diff
@@ -3,27 +3,21 @@ declare module "eslint-plugin-react" {
 
 	const configs: Record<"jsx-runtime" | "recommended", Required<Linter.Config>>;
 
-	export default {
-		configs,
-	};
+	export { configs };
```

Comment:
Try to not have unnecessary changes in your PR

Reply:
I did this because linter wouldn't let me make a commit

Reply:
I understand, but try to find out why did linter start complaining about that? Because previously it was okay, now suddenly it's not

Reply:
Strange, everything works fine now, but when pushing commit [f8b7660](https://github.com/BinaryStudioAcademy/bsa-2024-bebalance/pull/115/commits/f8b766023698543b7c69ff4b487cfe3af84687ec) the CI step for code linting failed with default exports